recipe/meta.yaml

```diff
@@ -470,7 +470,7 @@ outputs:
       commands:
         - test -f ${PREFIX}/lib/libasan.so
         - file ${PREFIX}/bin/{{ triplet }}-gcc
-        - echo 'void main(){}' | {{ triplet }}-gcc -fsanitize=address -x c -
+        - echo 'void main(){}' | {{ triplet }}-gcc -fsanitize=address -Wl,--fatal-warnings -x c -
```

Comment:
looks like libsanitizer needs to depend on `libstdcxx-ng`:
```
+ x86_64-conda-linux-gnu-gcc -fsanitize=address -Wl,--fatal-warnings -x c -
$PREFIX/x86_64-conda-linux-gnu/bin/ld: warning: libstdc++.so.6, needed by $PREFIX/lib/gcc/x86_64-conda-linux-gnu/11.4.0/libasan.so, not found (try using -rpath or -rpath-link)
collect2: error: ld returned 1 exit status
```

Reply:
Still fails the same way. It's possible that `$PREFIX/x86_64-conda-linux-gnu/bin/ld` isn't looking in `$PREFIX/lib` in this direct invocation.